coldfront/core/project/management/commands/add_default_project_choices.py

```diff
@@ -34,8 +34,8 @@ def handle(self, *args, **options):
             AttributeType.objects.get_or_create(name=attribute_type)
```

Comment:
Please remove all the `delete()` calls in this file. For example: `ProjectReviewStatusChoice.objects.all().delete()`